services/spar/src/Spar/Scim/User.hs

```diff
@@ -289,6 +307,12 @@ updateValidScimUser
   :: forall m. (m ~ Scim.ScimHandler Spar)
   => ScimTokenInfo -> UserId -> ValidScimUser -> m (Scim.StoredUser SparTag)
 updateValidScimUser tokinfo@ScimTokenInfo{stiIdP} uid newScimUser = do
+    -- TODO(arianvp):
+    --   getUsers currently synthesizes SCIM users out of thin air, but doesn't store
+    --   them in the SCIM table. Maybe it should. However one thing it for sure 
+    --   does not do is set the ManagedByScim flag, because get should be read-only
+    --   So PUT (which might be issued after a GET /Users) must set ManagedByScim
+    --   if this was not yet the case for this user
```

Comment:
see above: it's weird, but get should both set the managed-by flag and write the scim data to spar.